api/pushNotifications/CloudEmail.js

```diff
@@ -0,0 +1,55 @@
+
+/*
+#     CloudBoost - Core Engine that powers Bakend as a Service
+#     (c) 2014 HackerBay, Inc. 
+#     CloudBoost may be freely distributed under the Apache 2 License
+*/
+
+
+var q = require("q");
+
+module.exports = function() {   
+
+   /**
+    *Description : Send Email to all users in the selected aplication
+    *Params: 
+    *- Param secureKey: Secure key of System
+    *Returns:
+    -Success : success on emails sent
+    -Error : Error Data( 'Server Error' : status 500 )
+    */
+    global.app.post('/email/:appId/campaign', function (req, res) {
+        console.log("++++ Email Campaign ++++++");
+        try{
+            var appId = req.params.appId;
+            var appKey = req.body.key;
+            var emailBody = req.body.emailBody
+            var emailSubject = req.body.emailSubject
+
+            global.appService.isMasterKey(appId,appKey).then(function (isMasterKey) {
+                if (isMasterKey) {
+                    global.emailService.sendEmailsToAll(appId,emailBody,emailSubject).then(function(data){
```

Comment:
change method to sendEmail
Pass the isMasterKey variable in this function which will be used in customService